src/models.jl

```diff
@@ -10,7 +9,7 @@ mutable struct Model
     pressures::PressureFields
     G::SourceTerms
     Gp::SourceTerms
-    forcing  # ::Forcing  # No type so we can set to nothing while checkpointing.
+    forcing  # ::Forcing  # No type so we can set it to nothing while checkpointing.
```

Comment:
An alternative is to use `forcing::F` and make `F` a parameter of the model. Then `Model{A, <:Nothing}` is an 'unforced' model.

Reply:
Ah, but I see you want to dynamically set it to nothing?

Perhaps we can design a checkpointing functionality that checks to see if something is a function before saving it --- for the future.

Reply:
Yeah right now it's just the checkpointing that needs to _remove_ the `Forcing` struct to serialize then put it back in. Users might want to dynamically change the forcing too I guess.

Things would be simpler if we could serialize functions, but I agree: might be worth designing a better checkpointer.